config/recipes/beats/README.md

```diff
@@ -0,0 +1,39 @@
+# Beats Configuration Library
```

Comment:
I'm wondering how those examples will be referenced from the documentation on the website. I think there's value in listing all the examples explicitly in the docs (ie. copying this markdown file content, basically).

Reply:
[I tried to include the samples in the documentation a long time ago](https://github.com/elastic/cloud-on-k8s/pull/2656) but the docs build only checks out the `docs` directory so that wasn't successful. We'll either have to move the samples inside the docs directory or figure out the magic incantation required by the docs build to checkout more directories.

Reply:
I planned to copy it to the docs almost 1:1.